recipes/utfcpp/all/conandata.yml

```diff
@@ -11,27 +14,6 @@ sources:
   "3.2.5":
     url: "https://github.com/nemtrif/utfcpp/archive/v3.2.5.tar.gz"
     sha256: "14fd1b3c466814cb4c40771b7f207b61d2c7a0aa6a5e620ca05c00df27f25afd"
-  "3.2.4":
-    url: "https://github.com/nemtrif/utfcpp/archive/refs/tags/v3.2.4.tar.gz"
-    sha256: "fde21a4c519eed25f095a1cd8490167409cc70d7b5e9c38756142e588ccb7c7e"
-  "3.2.3":
-    url: "https://github.com/nemtrif/utfcpp/archive/v3.2.3.tar.gz"
-    sha256: "3ba9b0dbbff08767bdffe8f03b10e596ca351228862722e4c9d4d126d2865250"
```

Comment:
The 3.2.3 is being used by `lief`, `xlnt` and `antlr4-cppruntime`. We need to maintain it 😄

Reply:
@ErniGH 
Thanks for your information!
I reverted it.